test/unit/cxx_test_shmem_bitwise_reduce.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+ *  This test program is derived from a unit test created by Nick Park.
+ *  The original unit test is a work of the U.S. Government and is not subject
+ *  to copyright protection in the United States.  Foreign copyrights may
+ *  apply.
+ *
+ *  Copyright (c) 2021 Intel Corporation. All rights reserved.
+ *  This software is available to you under the BSD license below:
+ *
+ *      Redistribution and use in source and binary forms, with or
+ *      without modification, are permitted provided that the following
+ *      conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
```

Comment:
This doesn't look exactly like BSD-3C.  Shouldn't it mirror the license in the main LICENSE file?

Reply:
Same question/comment for all 3 tests.

Reply:
Ah, I'm not sure what exactly is going on here, but it looks like it's more than just `shmem_team_reduce.c` that is slightly inconsistent with the LICENSE file.  This text looks more like BSD-2C, but even that is not exactly it...

I think we should keep these tests and the previous tests as they are for now (because we're copying most of the original tests), but I will look into (i.e., ask John) whether we can/should change them to the BSD-3C we have in the main license file for SOS v1.5.3 (there's also the small matter that our main LICENSE file is actually variant SPDX:BSD-3C-Open-MPI).

Reply:
Actually, most unit tests are having a 2-clause statement.

Reply:
Right, but it's not the exact same BSD 2-clause as the SPDX version: https://spdx.org/licenses/BSD-2-Clause.html